full-service/src/db/txo.rs

```diff
@@ -376,10 +380,14 @@ impl TxoModel for Txo {
             // transaction), we need to update it
             Ok(txo) => {
                 txo.update_as_received(
-                    account_id_hex,
                     subaddress_index,
                     key_image,
                     received_block_index,
+                    account_id_hex,
+                    amount,
+                    &txo.target_key,
+                    &txo.public_key,
```

Comment:
So, these 2 should definitely be encoded as things currently work, and it's a bit concerning that they're passing tests without...

Reply:
Oh I see what's going on...